### ISSUE TITLE
WIndows Version 4.3 features not working 

### ISSUE THREAD

**Issue by KAnderson** [mfractor/mfractor-feedback#293]
I Installed 4.3 and went through all the features in the blog, On my project the following did not work as described.

Navigate to Binding Context 
Navigate to Static Resource
Integrated Help Links
C# Tooltips
* localisations
* colors
* images
* date formats

##Installation Information

```
--Platform Information--
Operating System: Windows
OS: Win32NT
OS Service Pack: 
OS Version: Microsoft Windows NT 10.0.19042.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.3.0.0
Product Variant: Visual Studio For Windows
Product Variant Version: 16.0.0.0
Product SKU: MFRACTOR-VS-WINDOWS
Commit SHA: 456a155ffb822243ae788e1ff912fa28be4af2e3
Revision: 4600
Branch: master
Date: 06/08/2020 2:26:45 AM (UTC)
Agent: Local
Installed extensions:
	
```

**Comment by matthewrdev** [comment]
Hi @KAnderson, thanks for the feedback.

Can you elaborate on "did not work as described"? Are they not appearing? Do the links not work?

**Comment by KAnderson** [comment]
I wasn't really sure if they were expected to work in Windows so I didn't take screenshots of everything but for instance, this is what I get when I try to do navigate to binding context
![image](https://user-images.githubusercontent.com/273611/89592831-dbf8a700-d80a-11ea-9393-08ee24cb2954.png)

**Comment by KAnderson** [comment]
similar for the other features, they just don't seem to be working, nothing for static resources, no help links, tool tips just aren't showing. The Xaml tooltips I am not sure because I didn't have any examples of such.

**Comment by KAnderson** [comment]
FYI, I did run the same project on VSMac and did see them there

**Comment by matthewrdev** [comment]
@KAnderson Regarding the `BindingContext.Score` image, dotted bindings are not yet supported (see #164).

**Comment by KAnderson** [comment]
Help links showed up suddenly but position is a little bit off 
![image](https://user-images.githubusercontent.com/273611/89658968-6df4c400-d88c-11ea-9fd0-4283832baffe.png)

**Comment by KAnderson** [comment]
Navigate to Binding Context is also working for simple cases but binding context within something like a CollectionView datatemplate gets confused - that might be a different issue entirely.

Navigate to Static Resources also works for some cases but not others. In the five static resources in this screenshot it works only for HeadingL and Colors.
![image](https://user-images.githubusercontent.com/273611/89659873-beb8ec80-d88d-11ea-9b48-78c7a0143423.png)

**Comment by KAnderson** [comment]
For C# Tooltips - color works, I was trying FromRGB which doesn't work but FromHex does. I didn't read close enough.

Hover over image does work but is the same as image analysis of names in xaml is case sensitive which means if you have the name not exactly matching case the analysis for images fails. In this case I had expand.png not Expand.png Also, not sure why the popup dialog is so wide...
![image](https://user-images.githubusercontent.com/273611/89662586-8a472f80-d891-11ea-998f-3a1f902e6871.png)

Date time format does work and I'm not sure about localizations, that shouldn't have been on my report.

**Comment by matthewrdev** [comment]
@KAnderson Could you provide a sample that reproduces where the collection view binding navigation is not working? The navigation engine uses the same infrastructure as the analysis engine, if one fails so will the other. It may be that there is an edge case that is not supported yet.

Regarding the other reports:

 * Image tooltip expansion: This is a bug.
 * Help Link position: This is a bug.
 * Navigation to static resource: I'll investigate, the tooltips between colors and normal resources are slightly different so it might need some cleanup.
 * Image casing: I'll reproduce this locally and investigate.

**Comment by matthewrdev** [comment]
@KAnderson I've reproduce the *image casing* bug report locally and on iOS, the casing for image assets is important.

**Example**
![image-asset-casing-sensitivity](https://user-images.githubusercontent.com/3260473/89844041-b7f7e580-dbbd-11ea-9896-c493250002f7.gif)

I have also tested this on a physical Android device and image names are also case sensitive.

Can you open a separate ticket for this issue with reproduction steps for the casing-sensitive bug?

**Comment by KAnderson** [comment]
> @KAnderson Could you provide a sample that reproduces where the collection view binding navigation is not working? 

I am not sure what has changed but this is working now?

**Comment by KAnderson** [comment]
> Can you open a separate ticket for this issue with reproduction steps for the casing-sensitive bug?

See #294

**Comment by matthewrdev** [comment]
Resolved in MFractor 4.3.1. To be released this week.
